packages/collections/_test.pony

```diff
@@ -752,3 +753,158 @@ class iso _TestHeap is UnitTest
+    h.assert_eq[U8](t.value(), 0b0001)
+
+    // Mutable Operator: difference
+    t = set0011.clone()              //0b0011
+    t.difference(set0101)            //0b0101
+    h.assert_eq[U8](t.value(), 0b0110)
+
+    // a.difference(b) == b.difference(a) **symetric difference**
+    t = set0101.clone()      //0b0101
+    t.difference(set0011)    //0b0011
+    h.assert_eq[U8](t.value(), 0b0110)
+
+    // Mutable Operator: remove
+    t = set0011.clone()      //0b0011
+    t.remove(set0101)        //0b0101
+    h.assert_eq[U8](t.value(), 0b0010)
+
+    // a.remove(b) != b.remove(a)
+    t = set0101.clone()      //0b0101
+    t.remove(set0011)        //0b0011
+    h.assert_eq[U8](t.value(), 0b0100)
+
+    // Mutable Operator: flip
+    t = set0011.clone()
+    h.assert_true(set0011 == t)                   //  t = 0b0011
+    t.flip(_TestFlagD)                            // flip 0b1000
+                                                  //      ------
+    h.assert_true(set0011 != t)                   //  t = 0b1011
+
+                                                  //  t = 0b1011
+    t.flip(_TestFlagD)                            // flip 0b1000
+                                                  //      ------
+    h.assert_true(set0011 == t)                   //  t = 0b0011
+
+
+    // Mutable Operator: all
+    t.all()
+    h.assert_true(t.apply(_TestFlagA) and
+                  t.apply(_TestFlagB) and
+                  t.apply(_TestFlagC) and
+                  t.apply(_TestFlagD))
+
+    // Mutable Operator: clear
+    t.clear()
+    h.assert_true((not t.apply(_TestFlagA)) and
+                  (not t.apply(_TestFlagB)) and
+                  (not t.apply(_TestFlagC)) and
+                  (not t.apply(_TestFlagD)))
+
+    // Mutable Operator: set
+    t .> set(_TestFlagA) .> set(_TestFlagD)
+    h.assert_true((    t.apply(_TestFlagA)) and // 1
+                  (not t.apply(_TestFlagB)) and // 0
+                  (not t.apply(_TestFlagC)) and // 0
+                  (    t.apply(_TestFlagD)))    // 1
+
+    // Check for idempotency
+    t .> set(_TestFlagA) .> set(_TestFlagD)
+    h.assert_true((    t.apply(_TestFlagA)) and // 1
+                  (not t.apply(_TestFlagB)) and // 0
+                  (not t.apply(_TestFlagC)) and // 0
+                  (    t.apply(_TestFlagD)))    // 1
+
+    // Mutable Operator: unset
+    t .> unset(_TestFlagD)
+    h.assert_true((    t.apply(_TestFlagA)) and // 1
+                  (not t.apply(_TestFlagB)) and // 0
+                  (not t.apply(_TestFlagC)) and // 0
+                  (not t.apply(_TestFlagD)))    // 0
+
+    // Check for idempotency
+    t .> unset(_TestFlagD)
+    h.assert_true((    t.apply(_TestFlagA)) and // 1
+                  (not t.apply(_TestFlagB)) and // 0
+                  (not t.apply(_TestFlagC)) and // 0
+                  (not t.apply(_TestFlagD)))    // 0
+
+type _TestFlagsFlags is Flags[(_TestFlagA|_TestFlagB|_TestFlagC|_TestFlagD),U8]
```

Comment:
I believe according the style guide there should be a space after the `,` so `, U8`.